easybuild/easyconfigs/c/Critic2/Critic2-1.1dev-foss-2023a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'Critic2'
+version = '1.1dev'
+
+homepage = 'https://aoterodelaroza.github.io/critic2/'
+description = """Critic2 is a program for the analysis of quantum mechanical
+calculation results in molecules and periodic solids."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'extra_fflags': '-ffree-line-length-none'}
+
+source_urls = ['https://github.com/aoterodelaroza/critic2/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['47c5b56ec370ae8f46b2248e8d0ed70b9d5b736b04ffe3936de5039d056ca785']
+
+configopts = '-DLIBCINT_INCLUDE_DIRS=$EBROOTLIBCINT/include/ '
+configopts += '-DLIBCINT_LIBRARY=$EBROOTLIBCINT/lib64/libcint.so '
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+]
+
+dependencies = [
+    ('libxc', '4.3.4'),  # use xc_f90_types_m and libxc_funcs_m need libxc < 5.0
```

Comment:
Hi, thank you for your contribution.

any reason to add an old libxc to the 2023a toolchain? It can create incompatibilities with other apps from the same toolchan that use newer libxc (maybe with a patch to Citric code)

I saw on citric2 webpage that the stable version based on citric1.1 is quite outdated - what do you think about upgrading to v 1.2? It should work with the more recent compilers and toolchains as is


From citric2 homepage:

> The development version works only with recent compilers (see [here](https://aoterodelaroza.github.io/critic2/installation/#whichcompilerswork)) but has the latest features.
> Version 1.2

Reply:
Hi, thank you for the comment. 
When I made the easyconfig, v1.2 was not released yet. The reason I created another libxc easyconfig for the latest toolchain is written in the comments. It depends on some old modules and it was not that easy to create a patch to replace all the functions.
I have created a new PR for Critic-1.2 now: https://github.com/easybuilders/easybuild-easyconfigs/pull/20833